### DISCUSSION TITLE
Disable delete button from release

### DISCUSSION THREAD

**Discussion by seffylmz** [community/community#45407] [New to GitHub]
Hi,

is it possible to disable the delete button for releases? This should apply to members with write roles.

![gh_release-delete-button](https://user-images.githubusercontent.com/15021901/214779081-41966654-4c56-4c98-b93d-2f09d9691027.png)

**Comment by VladCuciureanu** [comment]
This is not currently possible as you can see in the GitHub Docs [here](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization)

Edit: On further documentation, I found [this](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-peoples-access-to-your-organization-with-roles/managing-custom-repository-roles-for-an-organization) that might be of your help. That might be in enterprise only though

**Comment by seffylmz** [comment]
Thanks @VladCuciureanu 

Yes, it is possible to create custom roles. However, the permissions that can be selected are very limited. It does not seem possible to create a role which has the same permissions as the "Write" role, without the option to delete releases.

**Reply by VladCuciureanu** [reply]
Yes, but creating a 'Write' role without 'Create and Edit Releases' permissions is possible.
While it isn't what you want exactly, that would be one option to take into account.
